yvalenta/search.rb

```diff
@@ -0,0 +1,55 @@
+class SearchVowels
+
+  def initialize
+    vowels
+  end
+
+  def open_file
+    f = File.open("../text.txt", "r")
+    run(f)
+    f.close
+  end
+
+  def time_start(start)
+    struct = Struct.new(:start)
+    @time_s = struct.new(start)
+  end
+
+  def time_finish(finish)
+    struct = Struct.new(:finish)
+    @time_f = struct.new(finish)
+  end
+
+  private
+
+  def vowels
```

Comment:
according told me @pwalvarado , should not be initialized in the way you say it

what should I do?